Update command-line-tools.rst (#3429)
mattermost/docs: source/administration/command-line-tools.rst

.. note::
  Ensure you run the Mattermost binary as the ``mattermost`` user. Running it as ``root`` user (for example) may cause complications with permissions as the binary initiates plugins and accesses various files when running CLI commands. Running the server as ``root`` may result in ownership of the plugins and files to be overwritten as well as other potential permissions errors.
  
.. note::

> When running Mattermost with the configuration stored in the database you might need to pass the database connection string when running CLI commands which can do this as follows: 

Reviewer: ```suggestion
When running Mattermost and using the CLI with the configuration stored in the database you might need to pass the database connection string as follows: 
```
